### PR TITLE
Add moving menu and ui in game

### PR DESCRIPTION
**Pull request by s-kania**

### CODE REVIEW

File: VatSpace/main.lua [hunk 1]
```diff
@@ -66,9 +66,20 @@ local function startGame()
...
 
+local function tryAgain()
+    gv.setVariablesToStartValues()
+    asteroids.create()
+    cats.create()
+    ui.restartUI()
+    startTimer()
+    gv.game_state = 'PLAYING'
+end
+
```
Comment (GitGitarra) [inline VatSpace/main.lua:76]: Are you sure this function is nesesery? Basically fulfills exactly the same function as the startGame function so I can't see a reason why to not use startGame funtion to restart game.

File: VatSpace/ui.lua [hunk 2]
```diff
@@ -65,11 +70,12 @@ local function createSpeechForSpaceman()
...
 
 local function createSpaceman()
-    local morawiecki = display.newImage("resources/graphics/morawiecki.png", 10, 250)
+    local morawiecki = display.newImage("resources/graphics/morawiecki.png", 10, 400)
     morawiecki:scale(0.1, 0.1)
     morawiecki.fill.effect = "filter.pixelate"
     morawiecki.fill.effect.numPixels = 15
     morawiecki:addEventListener("touch", createSpeechForSpaceman)
+    transition.to( morawiecki, { time = 500, delay = 250, y = 250 } )
 end
```
Comment (GitGitarra) [inline VatSpace/ui.lua:78]: Moving elements on the beginning of the game play working and looking great.

**Review by GitGitarra — CHANGES_REQUESTED** [review]
The name of branch and commit is telling that feature is moving menu but I can see only elements in game play moving (this look great). Please pay attention to make more atomic commits :) 
After "try again" you can't see this moving elements anymore. Probably it's also because the background is create again but this should change after merge with master.

**Comment by GitGitarra** [comment]
And please resolve conflicts before pull request.

File: VatSpace/main.lua (hunk 1 above)
Comment (s-kania) [reply VatSpace/main.lua:76]: If we use startGame it will create menu again with moving features. Then it looks bad.

Comment (GitGitarra) [reply VatSpace/main.lua:76]: In my opinion it should always move when you start playing.

**Pull request merged**